parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -3979,6 +3979,8 @@ SqlNode BuiltinFunctionCall() :
         node = CaseN() { return node; }
     |
         node = RangeN() { return node; }
+    |
+        node = FirstLastValue() { return node; }
```

Comment:
Since we plan to parse FIRST_VALUE and LAST_VALUE using this function in Dialect1, let's remove "<FIRST_VALUE>" and "<LAST_VALUE>" from Dialect1's ReservedFunctionName() list.